### PR TITLE
Add option to delay building search index until it's needed

### PR DESCRIPTION
**Pull request by Glease**
A lot of people don't even use SFM anyway. For those who do care could just toggle this back.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# Do you need consistent player progressing (for example Thaumcraft)? -> Select a name
+# Do you need to test how your custom blocks interacts with a player that is not the owner? -> leave name empty
+developmentEnvironmentUserName = Developer
+
+# Define a source file of your project with:
+# public static final String VERSION = "GRADLETOKEN_VERSION";
+# The string's content will be replaced with your mods version when compiled. You should use this to specify your mod's
+# version in @Mod([...], version = VERSION, [...])
+# Leave these properties empty to skip individual token replacements
+replaceGradleTokenInFile = Reference.java
```
Comment (bombcar) [inline gradle.properties:29]: Build failed in GitHub and may because this needs tuning

Comment (bombcar) [reply gradle.properties:29]: Ah it’s mcmod.info needs modVersion instead of version

**Review by Dream-Master — APPROVED** [review]
